api/all.yaml

```diff
@@ -1414,6 +1702,39 @@ components:
           description: |
             Opaque token to retrieve the next page of results. Absent if there are no more pages.
             __page_token__ should be set to this value for the next request (for the next page of results).
+    GenerateTemporaryModelCredential:
+      example:
+        model_id: model_id
+        operation: null
+      properties:
+        model_id:
+          description: |
+            Model id for which temporary credentials are generated. 
+            Can be obtained from models/{full_name} (get model info) API.
```

Comment:
Don't think this is right

Reply:
Did we copy paste from the tables API? We should also cross-reference with our own openapi spec for models in UC REST APIs to make sure we have these details correct